pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go

```diff
@@ -150,6 +164,20 @@ func (qt *quotaTree) iterationForRedistribution(totalRes, totalSharedWeight int6
 	}
 }
 
+type SortQ struct {
```

Comment:
Maybe we need to add some UT to test this function.

Reply:
ok

Reply:
done